spec/lib/oops_request/states_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe 'OopsRequest::States' do
+  let(:request) { create :oops_request }
+
+  it 'have state pending' do
+    expect(request.state).to eq('pending')
+  end
+
+  context 'with an error' do
+    let(:message) { 'error from stub' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.